src/interaction/interactiveTarget.js

```diff
@@ -8,41 +8,39 @@
  * @example
  *      function MyObject() {}
  *
- *      Object.assign(
- *          MyObject.prototype,
- *          PIXI.interaction.interactiveTarget
- *      );
+ *      PIXI.interaction.interactiveTarget.call(MyObject.prototype)
  */
-export default {
+export default function ()
```

Comment:
Hmm hmm. Okay, I may need to do something more involved that Object.assign to apply the mixin then, since otherwise I can't think of an easy way to give each interactiveTarget a unique object to use with _trackedPointers